src/backend/wine/manager/ipc_handler.ts

```diff
@@ -19,12 +15,7 @@ ipcMain.handle('installWineVersion', async (e, release) => {
       progress
     })
   }
-  const result = await installWineVersion(
-    release,
-    onProgress,
-    createAbortController(release.version).signal
-  )
-  deleteAbortController(release.version)
+  const result = await installWineVersion(release, onProgress)
```

Comment:
since the abortId is `release.version`, I'm just reading that inside `installWineVersion` so no need of a parameter, but it's created the same way as before